pages/events/[id].js

```diff
@@ -1,34 +1,32 @@
 import { useRouter } from "next/router";
-import { events } from "@/resources/events";
-import { StyledCard } from "@/components/Card";
+import { StyledCard } from "../../components/Card";
+import Button from "../../components/Button";
 
-export default function EventDetails() {
+export default function EventDetails({ events }) {
   const router = useRouter();
   const { id } = router.query;
 
-  const goBack = () => {
-    router.push(`/`);
-  };
-
   const event = events.find((event) => event.id === id);
 
+  if (!event) {
+    return <h1>No event found.</h1>;
+  }
+
   return (
     <>
       <h1>Details</h1>
-      {event ? (
-        <StyledCard>
-          <h2>Name: {event.name}</h2>
-          <h2>Event: {event.event}</h2>
-          <h2>Date: {event.date}</h2>
-          <p>Location: {event.location}</p>
-          <p>Tasks: {event.tasks}</p>
-          <p>Ideas, Message, Thoughts etc: {event.ideas}</p>
-          <p>Guests: {event.guests}</p>
-        </StyledCard>
-      ) : (
-        <p>No event found.</p>
-      )}
-      <button onClick={goBack}>Go Back</button>
+      <StyledCard>
+        <h2>Name: {event.name}</h2>
+        <h2>Event: {event.event}</h2>
+        <h2>Date: {event.date}</h2>
+        <p>Location: {event.location}</p>
```

Comment:
instead of using <p> tags, you can use a list here and style that according to your preferences

Reply:
Thanks for the good suggestion and noted with thanks. I will definitely implement this in my next US or when I start proper styling.